src/App.js

```diff
@@ -1,17 +1,29 @@
+import React, { useEffect } from 'react';
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
 import LoginPage from './components/Pages/login/LoginPage';
 import MainPage from './components/Pages/main/MainPage';
+import ReservationForm from './components/Pages/reserve/Reserve';
+import { fetchDoctors } from './redux/doctors/DoctorList';
+
 import './App.css';
 
-const App = () => (
-  <div className="App">
+const App = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchDoctors());
+  }, []);
+
+  return (
+
```

Comment:
Please, kindly remove any changes you made on  this part of the project. You removed the styling who goes for a great part of the project.

Reply:
I think it's also necessary for loading data, Unfortunately I can't remove this part if you want to know more about this will meet and discuss